cui.cpp

```diff
@@ -215,6 +216,18 @@ void exit_ui ()
 
 void ui_tick ()
 {
+	fd_set fds;
```

Comment:
it might make ui_tick cleaner / more readable by wrapping this code in an aptly named helper function and calling that here.